core/server/src/main/webapp/browse-body.jsp

```diff
@@ -16,6 +16,33 @@
 <%@ page import="static java.net.URLEncoder.encode" %>
 
 <jsp:include page="header-scripts.jsp" />
+
+<script>
+    var currentDir = "<%= request.getAttribute("currentPath").toString()%>";
+    function getInputPath() {
+        var path = $.trim($("#pathInput").val());
+        return path;
+    }
+
+    function changeDir() {
+        var path = getInputPath();
+        path = encodeURI(path);
+        window.location.href="./browse?path="+path;
+    }
+
+    $(document).ready(function () {
+        $("#pathInput").val(currentDir);
```

Comment:
I think you mean line 34.
Because of i want to give a current path to the input tag which's id is pathInput.